.github/workflows/golang.yaml

```diff
@@ -49,7 +49,9 @@ jobs:
         args: -v --timeout 5m
         skip-cache: true
     - name: Check golang modules
-      run: make check-vendor
+      run: |
+        make check-vendor
+        make -C deployments/devel check-modules
```

Comment:
Question: Why don't we use the same target in all cases?

Reply:
OK. Should we then rename the target in `Makefile`?

Reply:
Sure. So make it `check-modules` everywhere?

Reply:
yes, let's do that.

Reply:
Done, please check now